frontend/src/assets/css/scaffolding.scss

```diff
@@ -33,6 +36,28 @@
   --supprot-error-color: #f76519;
 }
 
+// custom scrollbar
+html {
+  scrollbar-color: $brand-green-color $green-100-header-menu-background;
+  scrollbar-width: thin;
+}
+
+::-webkit-scrollbar {
+  width: 8px;
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: $brand-green-color;
+  box-shadow: inset 0 0 6px rgb(0 0 0 / 50%);
+}
+
+::-webkit-scrollbar-track {
+  background-color: $green-100-header-menu-background;
+  box-shadow: inset 0 0 6px rgb(0 0 0 / 40%);
```

Comment:
since we have a light theme too, box shadow should be also in a separate variables